tests/Dropdown/Bootstrap5Test.php

```diff
@@ -0,0 +1,405 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Widgets\Tests\Dropdown;
+
+use PHPUnit\Framework\TestCase;
+use Yiisoft\Definitions\Exception\CircularReferenceException;
+use Yiisoft\Definitions\Exception\InvalidConfigException;
+use Yiisoft\Definitions\Exception\NotInstantiableException;
+use Yiisoft\Factory\NotFoundException;
+use Yiisoft\Yii\Widgets\Dropdown;
+use Yiisoft\Yii\Widgets\Tests\Support\Assert;
+use Yiisoft\Yii\Widgets\Tests\Support\TestTrait;
+
+/**
+ * @psalm-suppress PropertyNotSetInConstructor
+ */
+final class Bootstrap5Test extends TestCase
+{
+    use TestTrait;
+
+    /**
+     * Add a header to label sections of actions in any dropdown menu.
+     *
+     * @link https://getbootstrap.com/docs/5.2/components/dropdowns/#headers
+     *
+     * @throws CircularReferenceException|InvalidConfigException|NotFoundException|NotInstantiableException
+     */
+    public function testHeaders(): void
+    {
+        $definitions = [
+            'containerClass()' => ['btn-group'],
+            'headerClass()' => ['dropdown-header'],
+            'headerTag()' => ['h6'],
+            'itemClass()' => ['dropdown-item'],
+            'itemsContainerClass()' => ['dropdown-menu'],
+            'toggleAttributes()' => [['aria-expanded' => 'false', 'data-bs-toggle' => 'dropdown']],
+            'toggleClass()' => ['btn btn-secondary dropdown-toggle'],
+        ];
+
+        Assert::equalsWithoutLE(
+            <<<HTML
+            <div class="btn-group">
+            <button type="button" class="btn btn-secondary dropdown-toggle" aria-expanded="false" data-bs-toggle="dropdown">Dropdown</button>
+            <ul class="dropdown-menu">
+            <li><h6 class="dropdown-header">Dropdown header</h6></li>
+            <li><a class="dropdown-item" href="#">Action</a></li>
+            <li><a class="dropdown-item" href="#">Another action</a></li>
+            </ul>
+            </div>
+            HTML,
+            Dropdown::widget($definitions)
+                ->items(
+                    [
+                        [
+                            'label' => 'Dropdown',
+                            'link' => '#',
+                            'items' => [
+                                ['label' => 'Dropdown header', 'link' => ''],
+                                ['label' => 'Action', 'link' => '#'],
+                                ['label' => 'Another action', 'link' => '#'],
+                            ],
+                        ],
+                    ]
+                )
+                ->render()
+        );
+    }
+
+    /**
+     * Similarly, create split button dropdowns with virtually the same markup as single button dropdowns, but with the
```

Comment:
Not sure if such descriptions are needed. We can see details using provided link.